packages/foam-core/src/model/graph.ts

```diff
@@ -27,7 +27,7 @@ export class FoamGraph implements IDisposable {
   /**
    * Maps the connections arriving to a URI
    */
-  public readonly backlinks: { [key: string]: Connection[] } = {};
+  public readonly backlinks: Map<string, Connection[]> = new Map();
```

Comment:
why was the change limited to `backlinks`, and not `links` and `placeholders`?

Reply:
The goal of the first commit was just to fix the bug.
`backlinks` and  `resourcesByName` are the only ones (Indexable Types) causing this bug.

But I am of the opinion to replace the other ones for consistency.